recipes/libtiff/all/conanfile.py

```diff
@@ -43,6 +43,8 @@ def build(self):
         cmake.definitions["lzma"] = False
         cmake.definitions["jpeg"] = False
         cmake.definitions["jbig"] = False
+        cmake.definitions["webp"] = False
+        cmake.definitions["zstd"] = False
```

Comment:
This also disables it for 4.0.9 and may break consumers

Why this change?

Reply:
This flag appears only in 4.1.0, therefore, it cannot affect 4.0.9.

I prefer to disable all dependencies that not specified in options because they can be accidentally picked up from system libraries on developer or CI machine.